client/components/navigation/Navbar.tsx

```diff
@@ -46,69 +53,172 @@ function Navbar() {
           </HStack>
         </Link>
 
-        <HStack spacing={{ base: 2, sm: 7 }}>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px"
-          href="/gigs">Gigs</Link>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          borderRadius="18px"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          href="/jobs">Jobs</Link>
+        <Button bg="gray.300" display={{ sm: 'block', md: 'none' }} zIndex={11}>
+          {isHamburgerOpen ? (
+            <CloseIcon
+              h={6}
+              w={6}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          ) : (
+            <HamburgerIcon
+              h={7}
+              w={7}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          )}
+        </Button>
```

Comment:
Chakra's [`IconButton` component](https://chakra-ui.com/docs/form/icon-button) would clean this up quite a bit!

```suggestion
        <IconButton 
          display={{ sm: 'block', md: 'none' }} 
          zIndex={11}
          onClick={hamburger.onToggle}
          icon={hamburger.isOpen ? <CloseIcon /> : <HamburgerIcon />}
        />
```